package.json

```diff
@@ -28,20 +28,27 @@
     "@bazel/typescript": "5.5.3",
     "@microsoft/api-extractor": "7.29.2",
     "@types/browser-sync": "^2.26.3",
+    "@types/license-checker": "^25.0.3",
     "@types/node": "16.10.9",
     "@types/selenium-webdriver": "^4.0.18",
     "@types/send": "^0.17.1",
+    "@types/spdx-satisfies": "^0.1.0",
     "@types/tmp": "^0.2.1",
     "@types/uuid": "^8.3.1",
     "@types/ws": "8.5.3",
     "@types/yargs": "^17.0.0",
     "browser-sync": "^2.27.7",
     "clang-format": "1.8.0",
+    "license-checker": "^25.0.1",
```

Comment:
All of the deps above are dev only dependencies and will be bundled by `ng-dev`.